src/lib/sfge/SfgeWrapper.ts

```diff
@@ -63,11 +63,12 @@ class SfgeSpinnerManager extends AsyncCreatable implements SpinnerManager {
 	}
 
 	public startSpinner(): void {
-		uxEvents.emit(EVENTS.START_SPINNER, `Evaluating SFGE rules. See ${this.logFilePath} for logs`, "Please wait");
+		uxEvents.emit(EVENTS.INFO_ALWAYS, `See ${this.logFilePath} for SFGE's logs`);
+		uxEvents.emit(EVENTS.START_SPINNER, `Analyzing with SFGE.`, "Please wait");
 
 		let intervalCount = 0;
 		this.intervalId = setInterval(() => {
-			uxEvents.emit(EVENTS.UPDATE_SPINNER, "Please wait." + ".".repeat(intervalCount));
+			uxEvents.emit(EVENTS.WAIT_ON_SPINNER, 'message is unused');
```

Comment:
You can remove everything with the interval, here. The point of this was to add a period to the end of the `please wait` string every 30 seconds. Now that we're adding more informative progress reports, this is redundant.

Reply:
The extra dots still look like something is happening especially when waiting on the next update. Let me know how you like it during QA. Based on that, we can keep it or remove it.